### ISSUE TITLE
screenshare doesn't appear on participants, only on the one who shares it

### ISSUE THREAD

**Issue by veenone** [jitsi/docker-jitsi-meet#875]
Hello,

Sorry if this a very basic question related to the infrastructure of jitsi meet.
currently I'm using build 5142-3 on my laptop, and everything is working (including screensharing).

however when I deploy this build (I'm transferring the docker images, and the same git repo I used on my laptop) to an isolated server (no internet, LAN only), services are OK except the screensharing.

whenever any participant sharing their screen, it doesn't get published to the others.
so screensharing only appear on the one who shares the screen, but seen by others as idle.

is there any suggestion where to start looking on the problems? 

Thank you.

**Comment by saghul** [comment]
Does regular video work across participants?

**Comment by veenone** [comment]
Hi @saghul,

Sorry for missing your reply for few months. I found out that the udp port range wasn't fully opened on the network.

It was working for some trials after the udp ports resolved. But the problem came back :(

I try to find any error information in the log but it  looks ok, no anomalies. But the fact screen sharing only shows up on my own session. Other participants don't see it.

What would be the best way to trace the problem?

I tried chrome v. 86 and firefox v. 83 both have same behavior
